src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/custom.py

```diff
@@ -1115,6 +1337,43 @@ def start_vmss(resource_group_name, vm_scale_set_name, instance_ids=None):
                                                        instance_ids=instance_ids)
 
 
+def list_vmss_instance_connection_info(resource_group_name, vm_scale_set_name):
```

Comment:
This was an internal command to list the ip address and port number for you to ssh to an instance, I have thought about deleting it, but CRP folks like to keep. I have pushed a few text change get it clear.